solr/solrj/src/test/org/apache/solr/client/solrj/io/stream/eval/ConversionEvaluatorsTest.java

```diff
@@ -32,7 +30,7 @@
 import org.junit.Test;
 
 /** Test ConversionEvaluators */
-public class ConversionEvaluatorsTest {
+public class ConversionEvaluatorsTest extends SolrTestCase {
```

Comment:
disagree

Reply:
Get all the benefits of the underlying randomization and thread leak checks from Lucene. This ensures that all these tests have a common base. There are no real downsides to doing this either.